pkgs/development/interpreters/love/0.9.nix

```diff
@@ -6,11 +6,11 @@
 
 stdenv.mkDerivation rec {
   pname = "love";
-  version = "0.9.1";
+  version = "0.9.2";
```

Comment:
This is still marked as broken. Did you test it?

Reply:
The source downloads and it tries to build but is still broken.

Reply:
Broken since 2019, should we just drop it and maybe also 0.7? I highly doubt someone is using these old versions.

Reply:
What do you mean «use»? I hope nobody writes new code for them, but old published game for those have to be run with a roughly correct version